src/encoded/static/components/form.js

```diff
@@ -11,6 +11,7 @@ import { FileInput, ItemPreview, ObjectPicker } from './inputs';
 import Layout from './layout';
 import DropdownButton from '../libs/ui/button';
 import { DropdownMenu } from '../libs/ui/dropdown-menu';
+import * as globals from './globals';
```

Comment:
ESLint ereror: You’re already importing from globals.js above, so you’ll need to resolve this.

Reply:
Fixed. Weird. Did not see this initially.